### PR TITLE
Adding scheduler

### PR DESCRIPTION
**Pull request by erez-work**

### CODE REVIEW

File: cloud_utils/scheduler/kubernetes_connector.py [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+                    "secret": {"secretName": secret["secret_name"]},
+                }
+            ],
+        ),
+    )
+
+
+make_deploy_cron_job = gamla.compose_left(
+    _get_kubernetes_client,
+    toolz.juxt(_create_cron_job),
```
Comment (nitzo) [inline cloud_utils/scheduler/kubernetes_connector.py:157]: looks like the juxt is redundant, you have only one function, also the compose left below

**Comment by erez-work** [comment]
I used CI_REGISTRY as repo identifier. Can you think of an alternative

File: cloud_utils/scheduler/kubernetes_connector.py [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+import os
+from pathlib import Path
+from typing import Dict, List, Text
+
+import gamla
+import toolz
+from kubernetes import client, config
+from kubernetes.client import rest
+
+repo_name = os.getenv("CI_REGISTRY")
```
Comment (nitzo) [inline cloud_utils/scheduler/kubernetes_connector.py:12]: this looks fine. but let's pass it as an argument to our tool

e.g. automation deploy $CI_REGISTRY

Comment (erez-work) [reply cloud_utils/scheduler/kubernetes_connector.py:12]: Sure, I'm also adding path to scheduling file as optional parameter

**Pull request merged**